packages/fast-components-react-base/src/index.ts

```diff
@@ -86,6 +86,10 @@ import SliderTrackItem from "./slider-track-item";
 export { SliderTrackItem };
 export * from "./slider-track-item";
 
+import StackPanel from "./stack-panel";
+export { StackPanel };
+export * from "./stack-panel";
```

Comment:
I'm not sure this should be available from the index location due to the note on this still being in initial development.

Reply:
good idea